packages/terra-table/src/subcomponents/ColumnHeaderCell.jsx

```diff
@@ -0,0 +1,220 @@
+import React, { useContext, useRef, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl } from 'react-intl';
+import * as KeyCode from 'keycode-js';
+import classNames from 'classnames/bind';
+
+import ThemeContext from 'terra-theme-context';
+import IconUp from 'terra-icon/lib/icon/IconUp';
+import IconDown from 'terra-icon/lib/icon/IconDown';
+import IconError from 'terra-icon/lib/icon/IconError';
```

Comment:
Can we used the updated import pattern? 

```suggestion
import {IconUp,IconDown,IconError } from 'terra-icon';
```

Reply:
So it turns out that I can't do this way because the way this package is set up. However we can do something like:

```import { IconUp, IconDown, IconError } from 'terra-icon/lib/IconExports';```